kbs/src/token/mod.rs

```diff
@@ -2,54 +2,86 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 
-use anyhow::*;
-use async_trait::async_trait;
+use jwk::JwkAttestationTokenVerifier;
+use kbs_types::TeePubKey;
 use serde::Deserialize;
-use std::sync::Arc;
+use serde_json::Value;
 use strum::EnumString;
-use tokio::sync::RwLock;
 
-mod coco;
+mod error;
 pub(crate) mod jwk;
+pub use error::*;
 
-#[async_trait]
-pub trait AttestationTokenVerifier {
-    /// Verify an signed attestation token.
-    /// Returns the custom claims JSON string of the token.
-    async fn verify(&self, token: String) -> Result<String>;
-}
+pub const TOKEN_TEE_PUBKEY_PATH_ITA: &str = "/attester_runtime_data/tee-pubkey";
+pub const TOKEN_TEE_PUBKEY_PATH_COCO: &str = "/customized_claims/runtime_data/tee-pubkey";
 
-#[derive(Deserialize, Default, Debug, Clone, EnumString)]
+#[derive(Deserialize, Default, Debug, Clone, EnumString, PartialEq)]
 pub enum AttestationTokenVerifierType {
     #[default]
     CoCo,
-    Jwk,
+
+    #[serde(rename = "ITA")]
+    Ita,
```

Comment:
this shouldn't be a user facing name/option.

Reply:
What name do you prefer to use?